prime-router/settings/organizations.yml

```diff
@@ -960,7 +960,6 @@
       customerStatus: active
       jurisdictionalFilter: [ "(%performerState.exists() and %performerState = 'CA') or (%patientState.exists() and %patientState = 'CA')" ]
       qualityFilter:
-        - "(%hl7MessageType.exists() and %hl7MessageType = 'R01')"
```

Comment:
Does this change need to be made in staging/prod?

Reply:
It could go either way, it's just repeated in another line that is part of the same filter below (gets cut off by the github view). Just was cleaning it up while I was in the area, but it's functionally the same either way.